tensorflow/core/kernels/tile_ops_impl.h

```diff
@@ -21,31 +21,8 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
```

Comment:
Because tile_functor do not need tile_ops_impl_*.cc to do specializations. I need to write a tile_ops_impl.cc to separate Tile and TileGrad. I think it would be more clear to separate them. Do you have a better idea?